### PR TITLE
DDS-1016 remove min chunk size validation

### PR DESCRIPTION
**Pull request by dmann**

### CODE REVIEW

File: api_design/DDS-974-enforce_upload_chunk_size.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The following properties will be added to the storage providers resource:
 `POST /projects/{id}/uploads`
 
 ###### Response Headers (Extensions)
-The following custom response headers will be added to inform clients of the minimum chunk size that may be utlized to ensure chunks can be coalesced, as well as the maximum chunk size the storage provider can accommodate.
+The following custom response headers will be added to inform clients of the minimum chunk size that may be utlized to ensure chunks can be coalesced when creating the maximum number of chunks, as well as the maximum chunk size the storage provider can accommodate.
```
Comment (dmlond) [inline api_design/DDS-974-enforce_upload_chunk_size.md]: maybe say "inform clients of a suggested minimum chunk size that can be utilized ..."

**Review by dmlond — APPROVED** [review]

File: api_design/DDS-974-enforce_upload_chunk_size.md (hunk 1 above)
Comment (dmann) [reply api_design/DDS-974-enforce_upload_chunk_size.md]: I don't understand what you want me to change here. Is it s/may/can/ ?

**Comment by dmann** [comment]
I think I'm going to have to kill this PR. I don't understand why it has merge conflicts...

File: api_design/DDS-974-enforce_upload_chunk_size.md (hunk 1 above)
Comment (dmlond) [reply api_design/DDS-974-enforce_upload_chunk_size.md]: I was suggesting the use of the word `suggested` :)
I just want to make sure people know that it is optional but recommended.

**Comment by dmlond** [comment]
:( so sad to see a good PR killed

**Comment by dmann** [comment]
Looks like I fixed it with a local merge, which didn't complain about merge conflicts. Not sure why github thought there were conflicts... :\

File: api_design/DDS-974-enforce_upload_chunk_size.md (hunk 1 above)
Comment (dmann) [reply api_design/DDS-974-enforce_upload_chunk_size.md]: Gotcha, that's a fair addition. I think I will also `s/can/will/` and `s/may/can/` while I'm at it.  :)

**Pull request merged**